packages/botonic-plugin-flow-builder/src/content-fields/whatsapp-button-list/flow-whatsapp-button-list-section.tsx

```diff
@@ -0,0 +1,35 @@
+import { HtWhatsappButtonListSection } from '../hubtype-fields'
+import { ContentFieldsBase } from './../content-fields-base'
+import { FlowWhatsappButtonListRow } from './flow-whatsapp-button-list-row'
+
+interface Row {
+  id: string
+  title: string
+  description: string
+}
+
+export class FlowWhatsappButtonListSection extends ContentFieldsBase {
+  public title = ''
+  public rows: FlowWhatsappButtonListRow[] = []
+
+  static fromHubtypeCMS(
+    component: HtWhatsappButtonListSection,
+    locale: string
+  ): FlowWhatsappButtonListSection {
+    const newButton = new FlowWhatsappButtonListSection(component.id)
+    newButton.title = ContentFieldsBase.getTextByLocale(locale, component.title)
+    newButton.rows = component.rows.map(row =>
+      FlowWhatsappButtonListRow.fromHubtypeCMS(row, locale)
+    )
+    return newButton
+  }
+
+  toBotonic() {
+    const rows = this.rows.reduce((acc: Row[], row) => {
+      const botonicRow = row.toBotonic() as Row
+      if (botonicRow.id) acc.push(botonicRow)
+      return acc
+    }, [])
```

Comment:
I think it may be clearer to use a `filter` and a `map` instead of a `reduce`.
```suggestion
    const rows = this.rows.filter(row => row.target).map(row => row.toBotonic)
```